Kernel/System/UnitTest/Selenium.pm

```diff
@@ -171,6 +171,11 @@ sub new {
     $Self->{BaseURL} = $Kernel::OM->Get('Kernel::Config')->Get('HttpType') . '://';
     $Self->{BaseURL} .= Kernel::System::UnitTest::Helper->GetTestHTTPHostname();
 
+    # Force usage of legacy webdriver methods in Chrome until things are more stable.
+    if ( lc $SeleniumTestsConfig{browser_name} eq 'chrome' ) {
+        $Self->{is_wd3} = 0;
+    }
```

Comment:
The Selenium image I chose was the last one that worked with ((OTRS)) CE. I believe it was the last one with a Chrome version that allowed some non-standard calls, but I can't really remember what i found out back then.

Newer Versions of Znuny had way fewer tests working (I don't have my notes anymore and I'm not sure about the details. Maybe it aborted the tests directly). I tried newer Selenium images, but those didn't work either. 

While looking through the differences made to the code for the tests, I found this change (removing the line I added here again) in the commit https://github.com/znuny/Znuny/commit/44b79b51c27eed906aa8dcfd41861d2ee9b770b7. Reverting it back  was what fixed most tests again, the other changes didn't have that effect.

I would have to assume that whoever added that change to the code tested it and got Selenium working somehow, but since I couldn't get it to work otherwise, adding the lines back was the solution I choose.

i.e. I did it because it worked.

Reply:
Thank you for the explanation.
Interesting, I didn't know that this code was already there once.
Maybe together we can find a stable configuration for the latest selenium versions.
I think we have made a good start with this. We have already updated some Selenium tests internally and got them running again.
Unfortunately, the "open" source from the former owner was no longer as transparent as we all expected.